docs/source/devguide/policies.rst

```diff
@@ -0,0 +1,35 @@
+.. _devguide_policies:
+
+========
+Policies
+========
+
+---------------------
+Python Version Policy
+---------------------
+
+OpenMC follows the Scientific Python Ecosystem Coordination guidelines `SPEC 0
+<https://scientific-python.org/specs/spec-0000/>`_ on minimum supported
+versions, which recommends that support for Python versions be dropped 3 years
+after their initial release.
```

Comment:
Minimum supported python right now is 3.8

https://github.com/openmc-dev/openmc/blob/dcb80338c5794490a99db58bbcdc7c25a9dd0439/setup.py#L64

Do we have any open tickets to bump to 3.10? Or make a note in the next release about the version jump?

Reply:
As of today, the Python API still works fine on 3.8 and 3.9, so there's not an immediate need to bump this. As we introduce code that _does_ require 3.9 or 3.10, we can bump this up accordingly... or we can bump it now (and update our CI testing matrix). I don't feel strongly either way.

Reply:
I personally think this should be bumped to `3.10`. I think not doing so can send a conflicting message. I personally read `python_requires` as all versions that match this are supported. Users might be confused when we say 3.8 isn't supported when `setup.py` says otherwise.

Reply:
@MicahGale I tend to agree. I just updated that and removed our Python 3.8 and 3.9 CI test configurations.